### PR TITLE
Regenerate with latest GIR

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by EPashkin** [comment]
Please update gir-files to `0bcaef96b` too.
👍 after this.

**Comment by sdroege** [comment]
Done :)

**Comment by EPashkin** [comment]
With regen please

**Comment by EPashkin** [comment]
Sorry, forgot refresh diff

**Comment by sdroege** [comment]
@GuillaumeGomez Please merge then :) It's also needed for https://github.com/gtk-rs/glib/pull/201

**Comment by GuillaumeGomez** [comment]
Thanks!

File: gobject-sys/src/lib.rs [hunk 1]
```diff
@@ -60,6 +60,10 @@ pub const G_VALUE_COLLECT_FORMAT_MAX_LENGTH: c_int = 8;
 pub const G_VALUE_NOCOPY_CONTENTS: c_int = 134217728;
 
 // Flags
+extern "C" {
+  pub fn g_binding_flags_get_type() -> GType;
```
Comment (GuillaumeGomez) [inline gobject-sys/src/lib.rs]: Arf, that's ugly! I skipped this part in the gir review. ><

**Comment by EPashkin** [comment]
@GuillaumeGomez What you prefer? One liner?

**Comment by GuillaumeGomez** [comment]
4 spaces indent. :)

**Comment by EPashkin** [comment]
Hm, I missed that too

**Comment by GuillaumeGomez** [comment]
This is a terrible situation (solved by https://github.com/gtk-rs/gir/pull/411). Emergency situation calls emergency PRs!

**Pull request merged**